.env.example

```
@@ -114,6 +114,21 @@ N8N_WEBHOOK_URL=
 
 # Sentry (error tracking)
 SENTRY_DSN=
+# Sentry REST API (used by management Excel — Engineering Health sheet)
+# Settings > Auth Tokens > Create New (scopes: project:read, event:read)
+SENTRY_AUTH_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_AUTH_TOKEN key should go before the SENTRY_DSN key

(UnorderedKey)